src/main/java/pages/page/factory/Engagement/ChallengesPage.java

```diff
@@ -0,0 +1,284 @@
+package pages.page.factory.Engagement;
+
+import elements.BaseElementFacade;
+import elements.TextBoxElementFacade;
+import net.serenitybdd.core.Serenity;
+import net.serenitybdd.core.annotations.findby.FindBy;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import pages.GenericPage;
+
+
+public class ChallengesPage extends GenericPage {
+    public ChallengesPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(xpath = "//*[@class='v-toolbar__content']//*[contains(@class,'d-lg-inline')]")
+    private BaseElementFacade addChallengeBtn;
+
+    public void checkAddChallengeBtn() {
+        addChallengeBtn.isVisibleAfterWaiting();
+    }
+
+    public void isAddChallengeBtnDisplayed() {
+        addChallengeBtn.shouldNotBeVisible();
+    }
+
+    @FindBy(xpath = "//*[contains(@class ,'v-list-item__content drawerTitle')]")
```

Comment:
We will need to change the xPath to introduce IDs using PRs on product, but this will be made in a second phase.